ui-tests/src/test/java/org/openmrs/reference/page/ClinicianFacingPatientDashboardPage.java

```diff
@@ -391,12 +391,17 @@ public String getPatientFamilyName() {
         return patientFamilyName;
     }
 
-    public String getTelephoneNumber() {
+    public String getTelephoneNumber() throws Exception{
         String phoneNumber = findElement(TELEPHONE_NUMBER_TEXT).getText();
         if (StringUtils.isNotBlank(phoneNumber)) {
-            phoneNumber = phoneNumber.substring(0, phoneNumber.indexOf("\n"));
+           for (int i = 0; i <=phoneNumber.length()-1; i++) {
+           for (int j = i+1 ; j <= phoneNumber.length(); j++) {
+           phoneNumber = phoneNumber.substring(0, phoneNumber.indexOf("\n"));
+           }
```

Comment:
I'm really confused about what this is supposed to do? Why do we have two variables? Where are `i` and `j` used?

Reply:
```suggestion
            phoneNumber = phoneNumber.trim();
```

Reply:
Sorry ,Will make it more meaningfull.

Reply:
Thanks @ibacher i had messed up recently